docs/proposed/hs_reassembly.md

```diff
@@ -0,0 +1,520 @@
+# Defragmenting handshake messages based on MPS reader
+
+# Summary
+
+At the momement, TLS handshake fragmentation is not supported. In particular
+this limits the max certificate length that mbedTLS client can receive.
+
+## Goals
+
+1. Full backwards compatibility with the mbedTLS 2.16 version, unless the user
```

Comment:
This should read "with development". 2.16 is an atavism, I will update this shortly.

Reply:
Addressed in 5d5d45f